react/react2/week1/github-users-search/src/components/GithubSearch.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, createContext } from "react";
+import "../App.css";
+import { GithubUser } from "./GithubUser";
+
+export const userContext = createContext(null);
+
+export function GithubSearch() {
```

Comment:
When there is more than one export function in a component, one should be the default one.
Here, userContext should be the default export function.

Reply:
give `{ children }` as props to `GithubSearch()`.